### PR TITLE
docs: update dashboard list visualization

### PR DESCRIPTION
**Pull request by mdcruz**
**What is this feature?**

Updates to the dashboard list visualization docs to include:
- how to configure a dashboard list visualization (video TBA)
- rearranging sections to match other visualization types

Also adding sections in text visualization and adding panel options shared file to flame graph
**NOTE**: Don't backport the Panel options shared file changes to versions 10.4 and 10.3

**Why do we need this feature?**

To explain to our users how to configure a dashboard list visualization easily and when to use it.

**Who is this feature for?**

Grafana OSS and Cloud users

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 1]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
 
 On each dashboard load, this panel queries the dashboard list, always providing the most up-to-date results.
 
+You can use a dashboard list visualization if you want to display a list of important dashboards that you want to track.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:31]:
```suggestion
You can use a dashboard list visualization to display a list of important dashboards that you want to track.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 2]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
 {{< docs/shared lookup="visualizations/panel-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
 
-## Options
+## Dashboard list options
+
+Use the following options to refine your dashboard list visualization.
+
+### Include current time range
+
+Select this option to propagate the time range of the current dashboard to the dashboard links. When the user clicks a link, the linked dashboard opens with the indicated time range already set.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:49]:
```suggestion
Select this option to propagate the time range of the current dashboard to the dashboard links. When you click a link, the linked dashboard opens with the indicated time range already set.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 3]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
+
+### Include current template variable values
+
+Select this option to include template variables currently used as query parameters in a link. When the user clicks the link, any matching templates in the linked dashboard are set to the values from the link. Learn more about [Dashboard URL variables][].
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:53]:
```suggestion
Select this option to include template variables currently used as query parameters in a link. When you click the link, any matching templates in the linked dashboard are set to the values from the link. Learn more in [Dashboard URL variables][].
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 4]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
-- **Recently viewed -** Display recently viewed dashboards in alphabetical order.
-- **Search -** Display dashboards by search query or tags. You must enter at least one value in **Query** or **Tags**. For the **Query** and **Tags** fields. Variable interpolation is supported, for example,`$my_var` or `${my_var}`.
-- **Show headings -** The chosen list selection (Starred, Recently viewed, Search) is shown as a heading.
-- **Max items -** Sets the maximum number of items to list per section. For example, if you left this at the default value of 10 and displayed Starred and Recently viewed dashboards, then the panel would display up to 20 total dashboards, ten in each section.
+Display recently viewed dashboards in alphabetical order.
 
-## Search
+### Search
+
+Display dashboards by search query or tags. You must enter at least one value in **Query** or **Tags**. For the **Query** and **Tags** fields, variable interpolation is supported. For example,`$my_var` or `${my_var}`.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:65]:
```suggestion
Display dashboards by search query or tags. You must enter at least one value in **Query** or **Tags**. For the **Query** and **Tags** fields, variable interpolation is supported. For example, `$my_var` or `${my_var}`.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 5]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
+
+### Show headings
+
+The chosen list selection (Starred, Recently viewed, Search) is shown as a heading.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:69]:
```suggestion
The selected list section (**Starred**, **Recently viewed**, **Search**) is shown as a  heading.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 6]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
+
+### Max items
+
+Sets the maximum number of items to list per section. For example, if you left this at the default value of 10 and displayed Starred and Recently viewed dashboards, then the panel would display up to 20 total dashboards, ten in each section.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:73]:
```suggestion
Sets the maximum number of items to list per section. For example, if you leave this at the default value of 10 and select **Starred** and **Recently viewed** dashboards, then the panel displays up to 20 total dashboards, 10 in each section.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 7]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
+## Search options
 
 These options only apply if the **Search** option is selected.
 
-- **Query -** Enter the query you want to search by. Queries are case-insensitive, and partial values are accepted.
-- **Folder -** Select the dashboard folders that you want to display.
-- **Tags -** Here is where you enter your tags you want to search by. Note that existing tags will not appear as you type, and they _are_ case sensitive.
+### Query
+
+Enter the query you want to search by. Queries are case-insensitive, and partial values are accepted.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:81]:
```suggestion
Enter the query by which you want to search. Queries are case-insensitive and partial values are accepted.
```

File: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md [hunk 8]
```diff
@@ -28,29 +28,65 @@ Dashboard lists allow you to display dynamic links to other dashboards. The list
...
+
+### Folder
+
+Select the dashboard folders that you want to display.
+
+### Tags
+
+Here is where you enter your tags you want to search by. Note that existing tags will not appear as you type, and they _are_ case sensitive.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/dashboard-list/index.md:89]:
```suggestion
Enter tags by which you want to search. Note that existing tags don't appear as you type, and they _are_ case sensitive.
```

**Review by imatwawana — COMMENTED** [review]
This looks great! I've made some wording suggestions to update the original language.

**Review by imatwawana — APPROVED** [review]

**Comment by mdcruz** [comment]
@imatwawana We've changed the YouTube published date to next week, the 20th of May because there'll be lots of videos published this week regarding G11. This PR can only be merged on or after the 20th. :)

**Pull request merged**